### DISCUSSION TITLE
Tooltip jumping issue in versions above 0.79.1

### DISCUSSION THREAD

**Discussion by Evirtual** [plouc/nivo#2448] [Q&A]
I spotted this issue just recently, but it is quit annoying. I am not experiencing this with 0.79.1 version. Only with later ones. 
[Video with an example](https://github.com/plouc/nivo/assets/4503621/dbed52b2-d698-4e31-8bf7-6e92ee893bcb). Is there a way to fix it?

**Comment by DimaAmega** [comment]
Hi @Evirtual!
I fixed it here in #2480
